Dockerfile

```diff
@@ -50,4 +51,4 @@ COPY JSON.lua /config/JSON.lua
 
 EXPOSE 8083
 
-ENTRYPOINT ["/entrypoint.sh"]
+ENTRYPOINT ["/usr/sbin/ot-recorder"]
```

Comment:
by removing `entrypoint.sh` the Recorder will not initialize its data store. That won't work.

Reply:
Done. I added it again.